File: Design-Patterns/Builder-Design-Pattern/java/src/dentist/SpecializationArea.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package dentist;

/**
 *
 * @author yousef
 */
public class SpecializationArea {
    private String areaOfSpecialization;
    private int startYear;

    public SpecializationArea(String areaOfSpecialization, int startYear) {
        this.areaOfSpecialization = areaOfSpecialization;
        this.startYear = startYear;
    }

    public String getAreaOfSpecialization() {
        return areaOfSpecialization;
    }

    public void setAreaOfSpecialization(String areaOfSpecialization) {
        this.areaOfSpecialization = areaOfSpecialization;
    }

    public int getStartYear() {
        return startYear;
    }

    public void setStartYear(int startYear) {
        this.startYear = startYear;
    }
    
    
}

File: Design-Patterns/Builder-Design-Pattern/php/Dentist/SpecializationArea.php
<?php
 

class SpecializationArea {
    private $areaOfSpecialization;
    private $startYear;

    public function __construct($areaOfSpecialization, $startYear) {
        $this->areaOfSpecialization = $areaOfSpecialization;
        $this->startYear = $startYear;
    }

    public function getAreaOfSpecialization() {
        return $this->areaOfSpecialization;
    }

    public function setAreaOfSpecialization($areaOfSpecialization) {
        $this->areaOfSpecialization = $areaOfSpecialization;
    }

    public function getStartYear() {
        return $this->startYear;
    }

    public function setStartYear($startYear) {
        $this->startYear = $startYear;
    }
    
    
}
